### PR TITLE
Create hearty.me.xml

### PR DESCRIPTION
**Pull request by chennien**

### CODE REVIEW

File: src/chrome/content/rules/hearty.me.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<ruleset name="hearty.me">
+	<target host="hearty.me" />
+	<target host="cdn.hearty.me" />
+	<target host="www.hearty.me" />
```
Comment (J0WI) [inline src/chrome/content/rules/hearty.me.xml]: * intro.hearty.me

**Comment by J0WI** [comment]
See also: #8047 #8076 #8077

**Comment by chennien** [comment]
@J0WI: I submitted these domains in separate xml files due to you asked me to do it separately. May I know what should I do next? Thanks a million.

**Comment by J0WI** [comment]
In this case these are just alias domains. So I'm fine with having them in one file.
Otherwise you should mention the related ruleset files in comments, that we are aware of them when we maintain the rules.

**Comment by J0WI** [comment]
I'm closing this, because this host is already preloaded in all major browser.

**Pull request closed without merging**